### DISCUSSION TITLE
kpanda/user-guide/gpu/nvidia/gpu-monitoring-alarm/gpu-alarm/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#5997] [Announcements]
# kpanda/user-guide/gpu/nvidia/gpu-monitoring-alarm/gpu-alarm/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/gpu/nvidia/gpu-monitoring-alarm/gpu-alarm/

**Comment by panpan0000** [comment]
很多客户要的是GPU故障的告警，而不仅仅是GPU利用率高的告警。

**Reply by lemontrees123** [reply]
gpu利用率只是举个例子，可以根据 XID 配置 GPU 故障的告警，操作方式是一样的
